assets/component-cart-items.css

```diff
@@ -108,16 +121,15 @@ cart-remove-button .icon-remove {
 }
 
 .cart-item .loading-overlay {
-  top: auto;
+  top: 0;
   left: auto;
-  right: 0;
+  right: auto;
   bottom: 0;
   padding: 0;
 }
```

Comment:
Wouldn't it be better to have the overlay centered and come up where the input number would usually be seen ?
[screenshot](https://screenshot.click/21-08-xbrlp-xeveq.png)

Reply:
cc. @melissaperreault what are your thoughts on this?

Reply:
Oh wow team I didn't realize this moved! Good catch again! Initially when removing an item, the loading was happening on the total column,  can we continue the same direction? [Video recording](https://screenshot.click/21-08-8qait-ntfc8.mp4)

Reply:
Is anyone able to reproduce this? I am having a hard time getting the spinner to show up and/or on the right element. I think this is happening more when there is more than one item in the cart.

https://user-images.githubusercontent.com/26778637/130473541-24e38fab-9a0d-4800-a31a-faa3fdcc7248.mov

Reply:
Thanks for finding this Martina. Can you try again now?